test-case/check-pause-resume-multi-mediaplayback-analog.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+##
+## Case Name: check-pause-resume-multi-mediaplayback-analog
+## Preconditions:
+##    N/A
+## Description:
+##    test playback with pause/resume on 2 separate simultaneous MediaPlayback
+##      & Analog pipelines
+##    fake pause/resume with expect
+##    expect sleep for sleep time then mocks spacebar keypresses ' ' to
+##    cause resume action
+## Case step:
+##    1. aplay on MediaPlayback pcm
+##    2. aplay on Analog/i2s pcm
+##    2. use expect to fake pause/resume in each pipeline
+## Expect result:
+##    no errors occur for either process
+##
+
+source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+
+OPT_OPT_lst['M']='pcm_MPb'      OPT_DESC_lst['M']='MediaPlayback pcm for playback. Example: hw:0,0'
+OPT_PARM_lst['M']=1             OPT_VALUE_lst['M']=''
+
+OPT_OPT_lst['A']='pcm_analog'   OPT_DESC_lst['A']='Analog pcm for playback. Example: hw:1,0'
+OPT_PARM_lst['A']=1             OPT_VALUE_lst['A']=''
```

Comment:
How about use TPLG to go though this value?

Reply:
I'm not sure I understand what you mean here? Are you saying pull this value from the tplg file?